src/Message.php

```diff
@@ -20,155 +25,227 @@
  */
 final class Message extends Widget
 {
+    private const COLORS = ['is-primary', 'is-link', 'is-info', 'is-success', 'is-warning', 'is-danger', 'is-dark'];
+    private const SIZES = ['is-small', 'is-medium', 'is-large'];
+    private array $attributes = [];
+    private string $autoIdPrefix = 'w';
     private string $body = '';
+    private array $bodyAttributes = [];
+    private array $buttonSpanAttributes = [];
+    private string $buttonSpanAriaHidden = 'true';
+    private string $buttonCssClass = 'delete';
+    private array $closeButtonAttributes = [];
+    private bool $encode = false;
+    private array $headerAttributes = [];
     private string $headerColor = 'is-dark';
     private string $headerMessage = '';
-    private array $options = [];
-    private array $bodyOptions = [];
-    private array $closeButtonOptions = [];
-    private array $headerOptions = [];
+    private string $messageBodyCssClass = 'message-body';
+    private string $messageCssClass = 'message';
+    private string $messageHeaderMessageCssClass = 'message-header';
     private string $size = '';
-    private bool $withoutCloseButton = false;
+    private bool $unclosedButton = false;
     private bool $withoutHeader = true;
 
     /**
-     * Returns a new instance with the specified message body.
+     * The HTML attributes. The following special options are recognized.
```

Comment:
There's nothing about options so "The following special options are recognized." isn't needed.

```suggestion
     * The HTML attributes.
```

Reply:
done.